### ISSUE TITLE
Duplicate units?

### ISSUE THREAD

**Issue by dr-shorthair** [qudt/qudt-public-repo#243]
unit:CAL_TH-PER-G
unit:CAL_TH-PER-GM

**Comment by steveraysteveray** [comment]
Good catch. We are using unit:GM for gram, and unit:G for "gravity", so the first one is not just a duplicate, it is an error. Where possible, we intend to align with UCUM for the choice of qname. In this case, one is g and the other is [g]. I think we made the right call here in using G for gravity and GM for gram.

You (or I) could remove the unit:CAL_TH-PER-G
@dr-shorthair, grab the assignment if you intend to do it.

**Comment by steveraysteveray** [comment]
Done.
